src/v/storage/tests/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ rp_test(
     offset_translator_state_test.cc
     file_sanitizer_test.cc
     compaction_reducer_test.cc
+    batch_consumer_utils_test.cc
```

Comment:
nit: if you prefer, gtest_storage exists as well. Or do the test utils require BOOST?

Reply:
The test is lightweight and dosn't require any mocking so it could be either boost or gtest.